packages/angular/src/utils/versions.ts

```diff
@@ -28,5 +28,6 @@ export const tsNodeVersion = '10.9.1';
 export const jestPresetAngularVersion = '~14.1.0';
 export const typesNodeVersion = '18.16.9';
 export const jasmineMarblesVersion = '^0.9.2';
+export const analogVitestAngular = '~1.7.2';
```

Comment:
Bumping this would be recommended, as it contains fixes for coverage reports and sourcemaps for the debugger.

```suggestion
export const analogVitestAngular = '~1.9.1';
```

Reply:
Thanks buddy!